packages/angular-material/test/group-layout.spec.ts

```diff
@@ -46,7 +45,7 @@ describe('Group layout', () => {
   beforeEach(() => {
     fixture = beforeEachLayoutTest(GroupLayoutRenderer, {
       declarations: [LayoutChildrenRenderPropsPipe, MatCard, MatCardTitle],
-      imports: [FlexLayoutModule],
+      imports: [],
```

Comment:
can this be removed now that it's empty?

Reply:
sure